File: src/main/java/com/movisens/xs/api/models/MonitoringFeedback.java
package com.movisens.xs.api.models;

import com.google.gson.annotations.Expose;

public class MonitoringFeedback extends Monitoring {
    @Expose
    public boolean sendAsInAppMessage = false;

    public MonitoringFeedback(long probandId, String date, MonitoringType type, String name,
                              String message, boolean includeInMail) {
        super(probandId, date, type, name, message, includeInMail);
    }
}

File: src/main/java/com/movisens/xs/api/models/MonitoringFeedback.kt
package com.movisens.xs.api.models

import com.google.gson.annotations.Expose

class MonitoringFeedback(
    probandId: Long,
    date: String,
    type: MonitoringType,
    name: String,
    message: String,
    includeInMail: Boolean
) : Monitoring(probandId, date, type, name, message, includeInMail) {
    @Expose
    var sendAsInAppMessage = false
}